ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
+
+__all__ = ["SensitivityPointSource"]
+
+
+def convert_astropy_array(arr, unit=None):
```

Comment:
Well, I do use the function in my other scripts, too, but since I didn't want to add my "helper_functions" collection here as a dependency, I simply copied the function definition.
The problem -- at the position where it is used -- is that `numpy.array` does not like lists of quantities (neither does matplotlib, btw), so the units have to be stripped and appended again to the np.array.
Anyway, I changed `make_mock_event_rate` to use `scipy.integrate` where I have to strip the units already in advance. So I no longer need `convert_astropy_array`.